src/e2e-test/features/spanner/source/SpannerToSpanner_Additional.feature

```diff
@@ -0,0 +1,118 @@
+@Spanner_Source @SPANNER_TEST
+Feature: Spanner source - Verification of Additional Spanner to Spanner successful data transfer without using connections
```

Comment:
remove **without using connection** text from the feature name as we already providing this details in the scenario name

Reply:
Noted